tests/test_lex.py

```diff
@@ -69,6 +69,12 @@ def test_lex_strings():
     assert objs == [HyString("abc")]
 
 
+def test_lex_strings_exception():
+    """ Make sure tokenize throws when codec can't decode some bytes"""
+    with lexe() as e: tokenize('\"\\x8\"')
```

Comment:
Do this across two lines *cough* *cough* PEP8 :wink: 

That said, I'd suggest renaming `e` to `execinfo`. Its consistent with other unit tests, and I think also better names what's provided as it isn't the exception itself.

Reply:
Ah, sorry I was just copying and pasting other part of this file...

Really should have a good read of PEP8, and maybe find some PEP8 plugin for vim...

Reply:
@waigx You can use `make flake` to run flake8 on the Hy codebase.